<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Table of content</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->






<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">dropEstAnalysis</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/VPetukhov/dropEstAnalysis">Source</a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Table of content</h1>

</div>


<div id="content-by-themes" class="section level2">
<h2>Content by themes</h2>
<ol style="list-style-type: decimal">
<li>Dataset annotations:
<ol style="list-style-type: decimal">
<li><a href="annotation__annotation_bmmc1.html">10x Frozen BMMCs (Healthy Control 1)</a></li>
<li><a href="annotation__annotation_pbmc8k.html">10x 8k PBMCs from a Healthy Donor</a></li>
<li><a href="annotation__annotation_bmc.html">inDrop Mouse Bone Marrow cells</a></li>
</ol></li>
<li>Origin of background cells in Human/mouse datasets:
<ol style="list-style-type: decimal">
<li><a href="human_mouse__hm_drop_seq.html">Drop-seq thousand</a></li>
<li><a href="human_mouse__hm_10x_1k.html">10x 1k 1:1 Mixture</a></li>
<li><a href="human_mouse__hm_10x_6k.html">10x 6k 1:1 Mixture</a></li>
</ol></li>
<li>Filtration of low-quality cells
<ol style="list-style-type: decimal">
<li><a href="low_quality_cells__label_noise_robustness.html">Robustness of classifier to noise</a></li>
<li><a href="low_quality_cells__validation_bmc.html">Validation of inDrop Mouse BMCs</a></li>
<li><a href="low_quality_cells__validation_mouse_pcs.html">Validation of inDrop Mouse PCs</a></li>
<li><a href="low_quality_cells__validation_bmmc1.html">Validation of 10x Human BMMCs</a></li>
<li><a href="low_quality_cells__validation_pbmc8k.html">Validation of 10x 8k PBMCs</a></li>
</ol></li>
<li>UMI corrections:
<ol style="list-style-type: decimal">
<li><a href="umi_correction__umi_bmmc1.html">Correction effect on 10x BMMCs dataset</a></li>
<li><a href="umi_correction__trim_aml_035_post.html">UMI trimming on 10x AML035 Post-transplant dataset</a></li>
</ol></li>
<li>Cell barcode merge validation
<ol style="list-style-type: decimal">
<li><a href="cell_barcode_correction__merge_targets_mixture.html">Merge of human/mouse mixtures</a></li>
<li><a href="cell_barcode_correction__size_gain.html">Number of molecules per cell</a></li>
</ol></li>
<li>Runtimes</li>
<li><a href="runtimes__indrop_esc.html">inDrop ESCs</a></li>
<li><a href="runtimes__indrop_bmc.html">inDrop BMCs</a></li>
</ol>
</div>
<div id="content-by-figures" class="section level2">
<h2>Content by figures</h2>
<p>Some figures were created with Python code and they are not published here. Please, write personally to request the code.</p>
<div id="main-figures" class="section level3">
<h3>Main figures</h3>
<ol style="list-style-type: decimal">
<li>Figure 1. Skewed distribution of UMIs leads to increased number of UMI collisions.
<ul>
<li><del>A. Skewness of UMI distribution</del></li>
<li><del>B. Uneven nucleotide frequencies</del></li>
<li><a href="umi_correction__trimmed_collisions.html#main_figure">C. Estimated number of collisions</a></li>
</ul></li>
<li><a href="umi_correction__trim_aml_035_post.html#complete_figure">Figure 2. Comparison of UMI collision and sequencing error correction methods.</a>
<ul>
<li><a href="umi_correction__trim_aml_035_post.html#figure_a">A. Collisions on trimmed UMIs</a></li>
<li><a href="umi_correction__trim_aml_035_post.html#figure_b">B. Collision adjustment for trimmed UMIs</a></li>
<li><a href="umi_correction__trim_aml_035_post.html#figure_c">C. Correction of UMI errors on trimmed data</a></li>
<li><a href="umi_correction__umi_bmmc1.html#edit_distances">D. Distribution of edit distances between pairs of UMIs</a></li>
<li><a href="umi_correction__umi_bmmc1.html#subset_for_main_figure">E. Magnitude of correction</a></li>
</ul></li>
<li>Figure 3. Correcting for Cellular Barcode errors.
<ul>
<li><a href="human_mouse__hm_drop_seq.html#figure">A. Origin of background cells</a></li>
<li><del>B. Number of equidistant adjacent CBs</del></li>
<li><a href="cell_barcode_correction__size_gain.html#figure">C. Increase in #molecules per cell</a></li>
</ul></li>
<li><a href="low_quality_cells__validation_bmmc1.html#final_figure">Figure 4. Selection of the optimal size threshold for 10x BMMCs dataset.</a>
<ul>
<li><a href="low_quality_cells__validation_bmmc1.html#plots">A. The distribution of molecular mass across CBs of different sizes</a></li>
<li><a href="low_quality_cells__validation_bmmc1.html#plots">B. Gene expression profiles of cluster-specific genes</a></li>
<li><a href="low_quality_cells__validation_bmmc1.html#plots">C. Gene expression profiles of cluster-specific genes</a></li>
<li><a href="low_quality_cells__validation_bmmc1.html#rescued_cells">D. t-SNE visualization</a></li>
</ul></li>
<li><a href="low_quality_cells__validation_pbmc8k.html#final_figure">Figure 5. Filtration of low-quality cells for the 10x 8k PBMCs dataset.</a>
<ul>
<li><a href="low_quality_cells__validation_pbmc8k.html#t-sne_with_noise">A. t-SNE visualization</a></li>
<li><a href="low_quality_cells__validation_pbmc8k.html#error_sources">B. Cells with high mitochondrial fraction</a></li>
<li><a href="low_quality_cells__validation_pbmc8k.html#error_sources">C. Cells with high intergenic fraction</a></li>
<li><a href="low_quality_cells__validation_pbmc8k.html#error_sources">D. Cells with low #UMIs per gene</a></li>
</ul></li>
<li><a href="low_quality_cells__validation_bmc.html#figure">Figure 6. Filtration of low-quality cells for the inDrop mouse BMCs dataset.</a>
<ul>
<li><a href="low_quality_cells__validation_bmc.html#heatmaps">A. Gene expression profiles of cluster-specific genes</a></li>
<li><a href="low_quality_cells__validation_bmc.html#heatmaps">B. Gene expression profiles of cluster-specific genes</a></li>
<li><a href="low_quality_cells__validation_bmc.html#rescued_cells">C. t-SNE visualization</a></li>
</ul></li>
</ol>
</div>
<div id="supplementary-figures" class="section level3">
<h3>Supplementary figures</h3>
<ol style="list-style-type: decimal">
<li><del>S1. Skewness of UMI distributions.</del></li>
<li>S2. Simulation of UMI collision frequencies.</li>
<li>S3. Probability of observing adjacent UMIs in small genes.</li>
<li><del>S4. Recognition of UMI errors by base calling quality.</del></li>
<li><a href="umi_correction__trimmed_collisions.html#complete_figure">S5. Impact of non-uniform distribution on UMI collisions</a></li>
<li><a href="umi_correction__trim_aml_035_post.html#supp_figure_collisions">S6. UMI collisions on trimmed data</a></li>
<li><a href="umi_correction__umi_bmmc1.html#normalized_expression">S7. Magnitude of UMI correction.</a></li>
<li><a href="umi_correction__trim_aml_035_post.html#supp_figure_corrections">S8. Comparison of UMI correction algorithms on trimmed data</a></li>
<li>S9. Initial labeling of high-quality cells based on cell size distributions.</li>
<li><a href="human_mouse__hm_10x_6k.html#figure">S10. Human and mouse cell mixture dataset by 10x</a></li>
<li>S11. Robustness of different classifiers to training errors.</li>
<li>S12. Annotation of the 10x Frozen BMMCs dataset.</li>
<li>S13. Annotation of the 10x 8k PBMCs dataset.</li>
<li><a href="annotation__annotation_bmc.html#expression_plots">S14. Annotation of the inDrop BMCs dataset.</a></li>
<li><a href="low_quality_cells__validation_pbmc8k.html#supplementary_figure">S15. Classification of low- and high-quality cells on 10x data.</a></li>
<li><a href="low_quality_cells__validation_pbmc8k.html#initial_labels_and_scores">S16. Comparison of the initial label assignments with the cell quality score predicted by the algorithm.</a></li>
<li>S17. Classification of low- and high-quality cells on inDrop mouse pancreatic cells data.</li>
</ol>
</div>
</div>
<div id="content-by-tables" class="section level2">
<h2>Content by tables</h2>
<div id="main-tables" class="section level3">
<h3>Main tables</h3>
<ol style="list-style-type: decimal">
<li>Table 1. Analysis of merge targets on human/mouse mixture datasets.</li>
<li>Table 2. 5-fold CV results (mean Â± sd).</li>
</ol>
</div>
<div id="supplementary-tables" class="section level3">
<h3>Supplementary tables</h3>
<ol style="list-style-type: decimal">
<li>Table S1. Gene markers, used for annotation of the 10x Frozen BMMCs.</li>
<li>Table S2. Gene markers, used for annotation of the 10x 8k PBMCs dataset.</li>
<li>Table S3. Gene markers, used for annotation of the inDrop mouse BMCs dataset.</li>
<li>Table S4. Fraction of rescued cells for the 10x Frozen BMMCs dataset.</li>
<li>Table S5. Fraction of rescued cells for the 10x 8k PMMCs dataset.</li>
<li>Table S6. Fraction of rescued cells for the inDrop mouse PCs dataset.</li>
<li>Table S7. Fraction of rescued cells for the inDrop mouse BMCs dataset.</li>
<li>Table S8. Runtimes of dropEst pipeline.
<ul>
<li>inDrop Mouse BMCs</li>
<li>inDrop Mouse ESCs</li>
</ul></li>
</ol>
</div>
</div>

<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>

<hr>
<p>
    This <a href="http://rmarkdown.rstudio.com">R Markdown</a> site was created with <a href="https://github.com/jdblischak/workflowr">workflowr</a>
</p>
<hr>

<!-- To enable disqus, uncomment the section below and provide your disqus_shortname -->

<!-- disqus
  <div id="disqus_thread"></div>
    <script type="text/javascript">
        /* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
        var disqus_shortname = 'rmarkdown'; // required: replace example with your forum shortname

        /* * * DON'T EDIT BELOW THIS LINE * * */
        (function() {
            var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
            dsq.src = '//' + disqus_shortname + '.disqus.com/embed.js';
            (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
        })();
    </script>
    <noscript>Please enable JavaScript to view the <a href="http://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
    <a href="http://disqus.com" class="dsq-brlink">comments powered by <span class="logo-disqus">Disqus</span></a>
-->



</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
